Makefile

```
@@ -0,0 +1,35 @@
+### ----- NOTES ----- ###
+# 
+### ----- NOTES ----- ###
+MAKEFLAGS += --no-print-directory
+WORK_DIR = $(shell pwd)
+OPS_SCRIPTS_DIR = "./ops/scripts"
+DBS_NAME = "dbs"
+APPS = ""
+ENV = "local"
```

```suggestion
OPS_SCRIPTS_DIR ?= "./ops/scripts"
DBS_NAME ?= "dbs"
APPS ?= ""
ENV ?= "local"
```

I think it might be worth changing these to assign only if not already defined," which allows users to override these values via environment variables or command-line arguments while providing sensible defaults.

https://www.baeldung.com/linux/makefile-conditional-variable-assignment-operator#:~:text=Using%20the%20Conditional%20Variable%20Assignment%20Operator%20(%3F%3D)&text=operator%2C%20which%20is%20used%20for,of%20using%20the%20login%20name.

---

This also lets us in Github actions to run these tasks with an environment block rather than having to set it after the make command.

i.e. 

```sh
APPS=apache-airflow make setup-apps # ✅ with this change
```

vs  currently written only allowing after the initial make variable has been loaded

```sh
make setup-apps APPS=apache-airflow
```